android/app/src/main/java/com/microsoft/codepush/common/exceptions/CodePushFileException.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.codepush.common.exceptions;
+
+/**
+ * Exception class for throwing exceptions occurred when working with files.
+ */
+public class CodePushFileException extends Exception {
```

Comment:
What is the reason for adding FileException if we already have IOException?

Reply:
@ruslan-bikkinin Ibelieve its no good to throw an `IOException`, we should throw our own exceptions only.

Reply:
I think I would change that in other places, too